sql/expression/aggregation_test.go

```diff
@@ -96,3 +97,53 @@ func TestFirst_Eval(t *testing.T) {
 	c.Merge(b, b2)
 	assert.Equal(int32(1), c.Eval(b))
 }
+
+func TestMin_Name(t *testing.T) {
```

Comment:
The same as MAX; can you add some tests with nulls please?

Corner cases here: http://www-cs-students.stanford.edu/~wlam/compsci/sqlnulls

Reply:
@ajnavarro since the NULL thing is an enhancement to be implemented #33, can I merge this PR?